treeherder/webapp/api/serializers.py

```diff
@@ -365,3 +366,12 @@ class MachinePlatformSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.MachinePlatform
         fields = ('id', 'platform')
+
+
+class ChangelogSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Changelog
+        # how do I grab files ? (one-to-many join)
```

Comment:
It depends on how it's returned in the API. Generally, you use a double underscore to access it in the 'fields' tuple but if you want it to be renamed nicely, you can use a `slugField` or `CharField`: https://github.com/mozilla/treeherder/blob/master/treeherder/webapp/api/serializers.py#L316